doc/release/upcoming_changes/14841.compatibility.rst

```diff
@@ -1,6 +1,11 @@
-Add ufunc loops to ``isinf`` and ``isnan`` for ``datetime64``, ``timedelta64``
-------------------------------------------------------------------------------
-Add needed infrastructure so ``np.isinf(a)`` and ``np.isnan(a)`` will run on
-``datetime64`` and ``timedelta64`` dtypes. This may require adjustment to user-
-facing code that either disallowed the calls or checked that they raised an
-exception.
+Add more ufunc loops for ``datetime64``, ``timedelta64``
+--------------------------------------------------------
+``np.datetime('NaT')`` should behave more like ``float('Nan')``. Add needed
+infrastructure so ``np.isinf(a)`` and ``np.isnan(a)`` will run on
+``datetime64`` and ``timedelta64`` dtypes. Also added specific loops for
+``fmin`` and ``fmax`` that mask ``NaT``. This may require adjustment to user-
+facing code. Specifically, code that either disallowed the calls to `isinf`` or
+``isnan`` or checked that they raised an exception will require adaptation, and
+code that mistakenly called ``fmax`` and ``fmin`` instead of ``maximum`` or
+``minimum`` respectively will requre adjustment. This also affects ``nanmax``
```

Comment:
I think nanmax raised before 1.18. (Or maybe I missunderstand this comment).

Reply:
fixing